pyproject.toml

```diff
@@ -13,8 +13,8 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.13"
-matchms = "^0.14"
+python = ">=3.10,<3.13"
+matchms = "0.28.2"
```

Comment:
Can we relax this dependency maybe a bit?

Reply:
@hechth, which of them would you like me to relax, better yet, kindly share an example.

Reply:
I mean the matchms one - pinning it only to 0.28.2 is maybe a bit strict - maybe >= 0.28.2?